readthedocs/builds/tests/test_views.py

```diff
@@ -105,3 +105,35 @@ def _get_project(self, owners, **kwargs):
 @override_settings(RTD_ALLOW_ORGANIZATIONS=True)
 class CancelBuildViewWithOrganizationsTests(CancelBuildViewTests):
     pass
+
+
+class BuildViewsTests(TestCase):
+    def setUp(self):
+        self.user = get(User, username="test")
+        self.project = get(Project, users=[self.user])
+        self.version = get(Version, project=self.project)
```

Comment:
You may need to set the privacy level of the project and version to public, so tests on .com pass.

Reply:
I started doing that, but then I realized that I will ignore these tests in .com because the spam concept doesn't make sense there -- rules are not even enabled.